client/src/pages/Menu.js

```diff
@@ -14,17 +15,25 @@ const MenuContainer = styled.div`
   }
 `;
 
-const menuOptions = [
-  'Client Master',
-  'ItemMaster',
-  'Storage System',
-  'Inbound',
-  'Outbound',
-];
+const menuRoutes = {
+  'Client Master': 'clientmaster',
+  'Item Master': 'itemmaster',
+  'Storage System': 'storgesystem',
+  Inbound: 'inbound',
+  Outbound: 'outbound',
+};
```

Comment:
with objects it's generally easier to change values rather than keys in the future. For example if you wanted to add another language and links in the menu should change accordingly.
I would propose to have this be an array and then change the map further down in line 33 accordingly. If you don't like that, you could also just turn the current object around, and have the keys as values and the values as keys, and change the map accordingly. I do however recommend using an array. 

```suggestion
const menuRoutes = [
  {
    route: 'clientmaster',
    label: 'Client Master'
  },
  {
    route: 'itemmaster',
    label: 'Item Master'
  },
  {
    route: 'storagesystem',
    label: 'Storage System'
  },
  {
    route: 'inbound',
    label: 'Inbound'
  },
  {
    route: 'outbound',
    label: 'Outbound'
  }
];
```
I hope this makes sense, let me know if it doesn't